pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -317,8 +317,14 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		if len(options.DNSIP) > 0 {
```

Comment:
are there old configs that just set DNSIP that will break (not start node DNS) if they upgrade and don't update their config to also set DNSBindAddress?